api/v1alpha1/common_types.go

```diff
@@ -43,9 +43,12 @@ type NetworkPolicyConfig struct {
 	// ClientLabels defines the pod selector clause that grants ingress access
 	// to the cluster client port(s).
 	ClientLabels map[string]string `json:"clientLabels,omitempty"`
-	// DashboardLabels defines the pod selector clause that grants ingress
-	// access to the cluster dashboard port.
-	DashboardLabels map[string]string `json:"dashboardLabels,omitempty"`
+	// Defines the pod selector clause that grants ingress
+	// access to the cluster dashboard.
+	DashboardPodLabels map[string]string `json:"dashboardPodLabels,omitempty"`
```

Comment:
I did test the upgrade path; since all of these labels are optional, a version mismatch between domino and the dco will just result in a more permissive netpol for resources created while the mismatch exists. Upgrading the dco will update the underlying crd's to have the new label fields instead of the old ones.

Reply:
Well, @sonnysideup says the way I did it would warrant a version bump, which feels like overkill for such a trivial PR. @ddl-kevin what do you think the change here should look like?

Reply:
Renamed `DashboardPodLabels` back to `DashboardLabels`